portfolio/src/main/webapp/script.js

```diff
@@ -40,6 +40,50 @@ async function deleteComments() {
   });
 }
 
+let map;
+let editMarker;
+
+/** Creates a map that allows users to add markers. */
+function initMap() {
+  const usaLat = 38.5949;
```

Comment:
consider defining these constants at the top of the file instead of inside the function.

Reply:
done.